website_sale_product_contract_gift/tests/test_gift.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+from odoo import Command, fields
+from odoo.tests import common
+
+
+class TestGiftContractBase(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.today = fields.Date.today()
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "partner test contract",
+                "email": "demo@demo.com",
```

Comment:
please use a valid example domain like `example.com`.

Reply:
Why ? It’s only generated for test.

Reply:
it’s not mandatory, indeed. i think it’s just good practice to use these reserved domain names for sample data, as it illustrates that they are only examples.

Reply:
then an url like @test.com would be better because it’s not for demo data but for tests.

Reply:
done.